tools/remote_admin/cmake/README.md

```diff
@@ -0,0 +1,28 @@
+# CMake Modules for RemoteAdmin
+
+This directory contains CMake modules copied from the RTI Connext DDS CMake Utilities repository for convenience. These modules are included locally to avoid requiring users to initialize git submodules.
```

Comment:
I think that we should keep it in a git submodule for maintainability reasons. We can add a note to the top-level readme that shows how to clone/initialize submodules properly. See an example here (section 'Installation'): https://github.com/rticommunity/rticonnextdds-gateway/tree/master

Also, in that example, we have the submodules aggregated in a top-level folder called third-party, so, they are available through the whole repo.